dist/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<!-- fontawesome -->
+		<script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+		<title>Movie App</title>
+	<script defer src="main.js"></script></head>
+	<body>
+		<header id="header"></header>
+		<main>
+			<h1>Trending Movies</h1>
+			<div class="movies"></div>
+		</main>
+		<footer>
+			<div class="footer-element">
+				<a
+              href="https://github.com/vickymarz"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Okoroji Victor Ebube</a
+            >
+            and
+            <a
+              href="https://github.com/usorfaitheloho"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Usor Faith Eloho</a
+            >
+			</div>
+		</footer>
+			<div class="popup-section"></div>
```

Comment:
I think you should consider moving the div above the footer as the footer signifies the end of the code. This is particularly good for semantic code.

Reply:
I have implemented the changes